_posts/2019-01-03-elixir-kaffe-codealong.md

```diff
@@ -0,0 +1,204 @@
+# Connecting Elixir to Kafka with Kaffe: A codealong
```

Comment:
We need to add the Jekyll meta data here at the top. You can copy an existing posts as a starting point.